package.json

```
@@ -39,5 +39,7 @@
     "eslint-plugin-jest": "^23.16.0",
     "eslint-plugin-mocha": "^7.0.1"
   },
-  "dependencies": {}
+  "dependencies": {
+    "ioredis": "^4.27.2"
+  }
```

This dependency seems unnecessary since we decided to use the `CacheRedis` framework from the Lisk Service Framework.

```suggestion
  "dependencies": {}
```